Hunting Queries/OracleDatabaseAudit/OracleDBAuditLargeQueries.yaml

```diff
@@ -0,0 +1,32 @@
+id: ba5e1a88-2054-4bda-a6e0-20008148ae6e
+name: OracleDBAudit - Users with new privileges
+description: |
+  'Query for searching user accounts whith new privileges.'
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+tactics:
+  - InitialAccess
+  - PrivilegeEscalation
+query: |
+  let lbtime_60d = 60d;

```

Comment:
30d max recommended as a default for hunting queries, but not required. Just remember this could cause performance issues.

Reply:
updated

Reply:
looks like this did not get updated, if you want to leave at 60d, just respond with that. :)